deploy/aks/monitoring/perf/prometheus-pushgateway-ingress.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: prometheus-pushgateway-ingress
   namespace: dapr-perf-metrics
   annotations:
+    nginx.ingress.kubernetes.io/use-regex: "true"
+    cert-manager.io/cluster-issuer: letsencrypt
```

Comment:
I am still baffled that we can't this isn't supported natively by Azure and that we have to even resort to manually setting up this ingress controller fronting with nginx, certificates with letsencrypt and all.